vampire.cpp

```diff
@@ -784,6 +779,9 @@ int main(int argc, char* argv[])
       exit(0);
     }
 
+    // can only happen after reading options as it relies on `env.options`
+    Timer::reinitialise();
+
```

Comment:
This has been lifted a little earlier during startup as I don't see why we wouldn't want to timeout during parsing. Happy to hear objections.

Reply:
We do want during parsing. Are you sure we didn't? But we don't want to time out during the interactive mode. Could you please check that's fine?

Reply:
Do I have it right that you determined this already and also fixed it in e583c89?

Reply:
Yes, we definitely had a bug since the interactive mode and making your PR compatible with the fix is probably already enough...